### ISSUE TITLE
throw error ' is not a multiple of (divisible by) 0.1'

### ISSUE THREAD

**Issue by HellowVirgil** [mozilla-services/react-jsonschema-form#342]
#### Actual behavior

I want to create a range widget which multiple of 0.1

```jsx
var JSONSchema = {
  "type": "object",
  "title": "Number fields & widgets",
  "properties": {
    "rangeSteps": {
      "title": "range",
      "type": "number",
      "minimum": 0,
      "maximum": 1,
      "multipleOf": 0.1
    }
  }
}
UISchema = {
  "rangeSteps": {
    "ui:widget": "range"
  }
}
```

When the value has been set as 0.6 or 0.3,the program throws the error 'instance.rangeSteps is not a multiple of (divisible by) 0.1'

**Comment by n1k0** [comment]
This is an issue with the [jsonschema] library and heavily resemble classic floating point arithmetic error:

```js 
> j = require("jsonschema")
{ Validator: [Function: Validator],
  ValidatorResult: [Function: ValidatorResult],
  ValidationError: [Function: ValidationError],
  SchemaError: [Function: SchemaError],
  validate: [Function] }
> s = {type: "number", multipleOf: 0.1}
{ type: 'number', multipleOf: 0.1 }
> test = x => j.validate(x, s).errors.map(e => e.message)
[Function]
> test(0)
[]
> test(0.1)
[]
> test(0.2)
[]
> test(0.3)
[ 'is not a multiple of (divisible by) 0.1' ]
> test(0.4)
[]
> test(0.6)
[ 'is not a multiple of (divisible by) 0.1' ]
> test(1)
[]
```

I've filed an issue on their bugtracker here https://github.com/tdegrunt/jsonschema/issues/187

**Comment by spacebaboon** [comment]
I've also just come up against this. It looks like there's some activity over on jsonschema in the last few days :smile:

**Comment by glasserc** [comment]
I believe this is closed now that #671 has been released.
